packages/commitlint-plugin-jira/src/rules/jira-project-key.ts

```diff
@@ -0,0 +1,32 @@
+import { Rule } from '@commitlint/types';
+
+export const JIRA_PROJECT_KEY = 'jira-project-key';
+
+export const jiraProjectKey: Rule<string[]> = ({ subject }, applicable, projects) => {
```

Comment:
As you mentioned in the description, I'm not sure why we are adding a plugin and config that we don't foresee using. As far I know, we encourage all teams to contribute to all projects... do you have a use case in mind for this?

Reply:
I definitely don't see us (BigCommerce) using it. But when I initially built it out I had in mind other consumers outside the organization potentially using it.

I guess the primary usage would be to check for the wrong spelling of a ticket (e.g. CHP vs CHIP).

Reply:
As I kind of hinted before, I would recommend not adding this to the repo, at least not at this point. We can add it later If we find a use for it.

Reply:
For sure, definitely more code we would need to maintain.